postgresql/build.sh

```diff
@@ -0,0 +1,27 @@
+# Cleanup directory
```

Comment:
Run this through [`checksec`](https://slimm609.github.io/checksec.sh/).

Reply:
Resolved with Commit 609e2845571188175dc916bdcdd1a707263587ed